jib-build-plan/src/main/java/com/google/cloud/tools/jib/api/buildplan/FileEntriesLayer.java

```diff
@@ -94,7 +93,7 @@ public Builder addEntry(Path sourceFile, AbsoluteUnixPath pathInContainer) {
       return addEntry(
           sourceFile,
           pathInContainer,
-          DEFAULT_FILE_PERMISSIONS_PROVIDER.apply(sourceFile, pathInContainer));
+          DEFAULT_FILE_PERMISSIONS_PROVIDER.getFilePermissions(sourceFile, pathInContainer));
```

Comment:
Now I think the name `getFilePermissions()` is redundant and we can just go with `FilePermissionsProvider.get()`. I've seen this naming pattern in many places. @loosebazooka WDYT?

Reply:
Yeah `get` makes sense to me.

Reply:
Lets put that change in here and then merge the PR.

Reply:
That's a good idea.